platform/ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -28,22 +60,31 @@ const Dropdown = ({ id, children, showDropdownIcon, list, titleClassName }) => {
             className="mr-2 w-4 text-white"
           />
         )}
-        <Typography className={titleClassName}>{title}</Typography>
+        <div
+          style={{
+            whiteSpace: 'nowrap',
+          }}
+        >
+          {title.length > maxCharactersPerLine && (
+            <div>
+              {lines.map((line, index) => (
+                <Typography
+                  key={index}
+                  className={itemsClassName}
+                >
+                  {line}
+                </Typography>
+              ))}
+            </div>
+          )}
+          {title.length <= maxCharactersPerLine && (
+            <Typography className={itemsClassName}>{title}</Typography>
+          )}
+        </div>
       </div>
     );
   }, []);
```

Comment:
I think the `useCallback` dependency should included (at least) `maxCharactersPerLine`, but I suppose if it does not change then it might not matter.